crates/ika/src/ika_commands.rs

```diff
@@ -57,7 +57,7 @@ use tempfile::tempdir;
 use tokio::runtime::Runtime;
 use tracing::info;
 
-const DEFAULT_EPOCH_DURATION_MS: u64 = 1000 * 60 * 10;
+const DEFAULT_EPOCH_DURATION_MS: u64 = 1000 * 150;
```

Comment:
revert

Reply:
its @ItayLevyOfficial's snapshot commit, idk how to fix this